src/portfolios/portfolios.service.ts

```diff
@@ -78,4 +80,20 @@ export class PortfoliosService {
       transactionBaseDto
     );
   }
+
+  public async updatePortfolioName(
+    portfolioParams: PortfolioDto,
+    params: ModifyPortfolioDto
+  ): ServiceReturn<NumberedPortfolio> {
+    const { did, id } = portfolioParams;
+
+    if (id.toNumber() <= 0) {
```

Comment:
```suggestion
    if (id.lte(0)) {
```
(nit) BigNumber has compare functions to use